bin/subiquity-tui

```diff
@@ -71,6 +71,7 @@ def parse_options(argv):
     parser.add_argument('--script', metavar="SCRIPT", action='append', dest='scripts', default=[], help='Execute SCRIPT in a namespace containing view helpers and "ui"')
     parser.add_argument('--click', metavar="PAT", action=ClickAction, help='Synthesize a click on a button matching PAT')
     parser.add_argument('--answers')
+    parser.add_argument('--snaps-from-examples', action='store_true', help="Load snap details from sample data, not store")
```

Comment:
Is this loading a directory of files? or is there a single index file?

--snaplist  matches closer to the --machine argument where we feed an input file.